source/_components/geofency.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Geofency"
+description: "Instructions for how to use Geofency to track devices in Home Assistant."
+date: 2017-08-22 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: geofency.png
+ha_category: Hub
```

Comment:
@arsaboo According to your above comment, then this should change since the linked page is categorized as a "Hub".

Reply:
This makes sense, it's not really a hub. I'm not sure what category it belongs in. I don't think both should have the same category, should they?

Reply:
I think that will also change then.

Reply:
Many of our current components do not list `ha_category`. Also, yes, they don't have to have the same category. For the platform, we should `Presence Detection` as category. I am not sure about the component.

Reply:
Can we not use `Presence Detection` for the component as well? Geofency is not being (and unlikely to be) used for anything else.